packages/react-core/src/components/Alert/examples/DangerAlert.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Alert, Button } from '@patternfly/react-core';
+import getContainerProps from './common/getContainerProps';
+
+class DangerAlert extends React.Component {
+  static title = 'Danger Alert';
+  static getContainerProps = getContainerProps;
```

Comment:
For consumers scss or react styles will work. I didn't want to be super opinionated about it for other projects. However, I would say react styles, but I am biased. Once we start including the code samples on the site this and some doc specific things will be stripped off to avoid confusion.

Reply:
To follow up. react-styles is already a dependency of react-core so that will happen either way. For react-tokens it is optional for now. Once we use it in react-core we will require that as well. These examples actually get copied over to docs and get published un `@patternfly/react-docs` so the dependencies there will drive what can be used in examples or `.docs.js` I have setup eslint to error on references to deps that do not exist within those files to make sure that contributors don't reference something incorrectly.